content/800-guides/1300-using-prisma-on-docker.mdx

```diff
@@ -0,0 +1,504 @@
+---
+title: "Using Prisma in Docker"
+metaTitle: "Using Prisma in Docker"
+description: "Learn step-by-step configure a Prisma ORM app in Docker"
+sidebar_label: "Using Prisma in Docker"
+image: "/img/guides/prisma-orm-docker.png"
+tags:
+  - Docker
+  - Alpine
+  - Containerization
+---
+
+This guide walks you through setting up a Prisma ORM application within a Docker environment. You’ll learn how to configure a Node.js project, integrate Prisma for database management, and orchestrate the application using Docker Compose. By the end, you’ll have a fully functional Prisma application running in a Docker container.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) installed
+- Node.js version: A [compatible Node.js version](/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-6#minimum-supported-nodejs-versions), required for Prisma 6.  
+
+Before starting, ensure that no PostgreSQL services are running locally, and that the following ports are free to avoid conflicts: `5432` (PostgreSQL), `3000` (application server) or `5555` (Prisma Studio server).
+
+To stop existing PostgreSQL services, use:
+
+```terminal
+sudo systemctl stop postgresql  # Linux
+brew services stop postgresql   # macOS
+net stop postgresql             # Windows (Run as Administrator)
+```
+
+To stop all running Docker containers and free up ports:
+
+```terminal
+docker ps -q | xargs docker stop
+```
+
+## 1. Set up your Node.js and Prisma application
+
+Let’s start by creating a simple Node.js application with Prisma ORM and [Express.js](https://expressjs.com/).
+
+### 1.1 Initialize your project
+
+First, create a new project directory and initialize a Node.js project:
+
+```terminal
+mkdir docker-test
+cd docker-test
+npm init -y 
+```
+
+This will generate a `package.json` file:
+
+```json file=package.json
+{
+  "name": "docker-test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {},
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+```
+
+### 1.2 Install required dependencies
+
+Next, install the Prisma CLI as a development dependency and Express.js for the server:
+
+```terminal
+npm install prisma --save-dev
+npm install express
+```
+
+### 1.3 Set up Prisma ORM
+
+Now, initialize Prisma to generate the necessary files:
+
+```terminal
+npx prisma init
+```
+
+This creates:  
+
+- A `prisma` folder containing `schema.prisma`, where you will define your database schema.  
+- An `.env` file in the project root, which stores environment variables.  
+
+Add a `User` model to the `schema.prisma` file located in the `prisma/schema.prisma` folder:
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+  //add-start
+  output = "../generated/prisma_client"
```

Comment:
I would like to see a note about this line with a link to: https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/generating-prisma-client#using-a-custom-output-path